doc/docs/master/concepts/pipeline-concepts/datum/relationship-between-datums.md

```diff
@@ -1,164 +1,86 @@
-
-In the diagram below, you can see the following datums:
-
- - `datum 1` has one file and results in one file in one output datum.
- - `datum 2` has one file and results in one file in one output datum.
- - `datum 3` has one file and results in one file in one output datum.
-
-![One to one](../../../assets/images/d_datum_processing_one_to_one.svg)
-
-If you decide to overwrite a single line in the file in `datum 3` and
-add `datum 4`, Pachyderm sees the four datums and checks them for changes
-one-by-one. Pachyderm verifies that there are no changes in `datum 1` and
-`datum 2` and skips these datums. Pachyderm detects changes in the
-`datum 3` and the `--overwrite` flag and replaces the `datum 3` with the
-new `datum 3'`. When it detects `datum 4` as a completely new datum,
-it processes the whole datum as new. Although only two datums were
-processed, the output commit of this change contains all four files.
-
-![One to one overwrite](../../../assets/images/d_datum_processing_one_to_one_overwrite.svg)
-
-## Example 2: One file in the input datum, multiple files in the output datum
-
-Some pipelines ingest one file in one input datum and create multiple
-files in the output datum. The files in the output datums might need to
-be appended or overwritten with other files to create the final commit.
-
-If you apply changes to that datum, Pachyderm does not detect which
-particular part of the datum has changed and processes the whole datum.
-In the diagram below, you have the following datums:
-
-- `datum 1` has one file and results in files `1` and `2`.
-- `datum 2` has one file and results in files `2` and `3`.
-- `datum 3` has one file and results in files `1` and `3`.
-
-![One to many](../../../assets/images/d_datum_processing_one_to_many.svg)
-
-Pachyderm processes all these datums independently, and in the end, it needs
-to create a commit by combining the results of processing these datums.
-A commit is a filesystem that has specific constraints, such as duplicate
-files with the same file path. Pachyderm merges results from
-different output datums with the same file path into single files. For
-example, `datum 1` produces `pfs/out/1` and `datum 3` produces `pfs/out/1`.
-Pachyderm merges these two files by appending them one to another
-without any particular order. Therefore, the file `1` in the final
-commit has parts from `datum1` and `datum2`.
-
-If you decide to create a new commit and overwrite the file in `datum 2`,
-Pachyderm detects three datums. Because `datum 1` and `datum 3` are
-unchanged, it skips processing these datums. Then, Pachyderm detects
-that something has changed in `datum 2`. Pachyderm is unaware of any
-details of the change. Therefore, it processes the whole `datum 2`
-and outputs the files `1`, `3`, and `4`. Then, Pachyderm merges
-these datums to create the following final result:
+### 1- **Creation of input datums** 
+In this stage, Pachyderm **breaks
```

Comment:
Calling it breaking isn't really correct. I would just say `Pachyderm creates datums based on the input data according to the...`